File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/Features.java
package com.philips.cdp.prxclient.datamodels.specification;

import java.io.Serializable;
import java.util.List;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Features implements Serializable {

	private static final long serialVersionUID = 4714472613554904424L;
	@SerializedName("feature")
	@Expose
	public List<FeatureItem> feature;
}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/Features.kt
package com.philips.cdp.prxclient.datamodels.specification

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Features : Serializable {
    @SerializedName("feature")
    @Expose
    var feature: List<FeatureItem>? = null

    companion object {
        private const val serialVersionUID = 4714472613554904424L
    }
}